RXSwiftTest/Model/Transaction.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Transaction.swift
+//  RXSwiftTest
+//
+//  Created by Sunith B on 30/08/19.
+//  Copyright © 2019 Sunith B. All rights reserved.
+//
+
+import UIKit
+
+class Transaction: Codable {
+    
+    //MARK: - Intance Variables
+    var id = ""
```

Comment:
Having properties with initial values as empty strings is not good practice. It's better to create a faillable initialiser so that you don't create broken objects that have empty properties.